### PR TITLE
Integrate kn-protocol FarmV2

### PR DESCRIPTION
**Pull request by viet-nv**

### CODE REVIEW

File: src/pages/MyEarnings/ElasticPools/SinglePosition/CollectFeesPanel.tsx [hunk 1]
```diff
@@ -131,6 +135,7 @@ const CollectFeesPanel: React.FC<Props> = ({
   }
 
   const collectFeeFromFarmContract = async () => {
+    console.log(farmAddress, feeValue0, feeValue1)
```
Comment (nguyenhoaidanh) [inline src/pages/MyEarnings/ElasticPools/SinglePosition/CollectFeesPanel.tsx:138]: rm

File: .env.adpr [hunk 2]
```diff
@@ -1,45 +1,44 @@
-VITE_AGGREGATOR_API=https://aggregator-api.dev.kyberengineering.io
```
Comment (nguyenhoaidanh) [inline .env.adpr:1]: todo

Comment (viet-nv) [reply .env.adpr]: reverted

File: src/state/farms/elasticv2/types.ts [hunk 3]
```diff
@@ -27,8 +27,8 @@ export interface ElasticFarmV2 {
   token0: Currency
   token1: Currency
   tvl: number
-  tvlToken0: CurrencyAmount<Currency>
-  tvlToken1: CurrencyAmount<Currency>
+  // tvlToken0: CurrencyAmount<Currency>
+  // tvlToken1: CurrencyAmount<Currency>
```
Comment (namgold) [inline src/state/farms/elasticv2/types.ts:31]: rm?

**Review by namgold — DISMISSED** [review]

File: src/state/farms/elasticv2/types.ts (hunk 3 above)
Comment (viet-nv) [reply src/state/farms/elasticv2/types.ts:31]: done

File: src/state/farms/elasticv2/updater.tsx [hunk 4]
```diff
@@ -107,37 +108,61 @@ export default function ElasticFarmV2Updater({ interval = true }: { interval?: b
   const { networkInfo, isEVM, chainId, account } = useActiveWeb3React()
   const elasticFarm = useAppSelector(state => state.elasticFarmV2[chainId] || defaultChainData)
   const { elasticClient } = useKyberSwapConfig()
+  const isEnableKNProtocol = true
```
Comment (namgold) [inline src/state/farms/elasticv2/updater.tsx:111]: is this temporary? @viet-nv

Comment (viet-nv) [reply src/state/farms/elasticv2/updater.tsx:111]: LOL. That's for testing only. You guys missed it when review the code. LMaO

Comment (viet-nv) [reply src/state/farms/elasticv2/updater.tsx:111]: Please help me to clear it

Comment (viet-nv) [reply src/state/farms/elasticv2/updater.tsx:111]: Thanks a lot

Comment (namgold) [reply src/state/farms/elasticv2/updater.tsx:111]: image

Comment (namgold) [reply src/state/farms/elasticv2/updater.tsx:111]: image

**Pull request merged**